### ISSUE TITLE
White screen when story has no design config

### ISSUE THREAD

**Issue by Romanchuk** [pocka/storybook-addon-designs#29]
Storybook: 5.3.8
storybook-addon-designs: 5.1.1

Exception occurs when you click on Design panel if story has no design config (user sees white screen). 

> Uncaught TypeError: Cannot use 'in' operator to search for 'length' in _undefined_

Group of stories decorated "withDesign", but story some stories has no design config.

```typescript
storiesOf('Controls', module)
	.addDecorator(withKnobs({ escapeHTML: false }))
	.addDecorator(withThemes(THEMES))
	.addDecorator(withDesign)
	.add('Checklists', data => {
		return {
..
```
http://joxi.ru/D2PKWaaSqxPGzm

I suggest to hide "Design" button if there is no design config in story or handle this exception to avoid braking the whole storybook.

**Comment by Romanchuk** [comment]
i see in code you have:
> skipIfNoParameterOrOptions: true
But no code references to it

**Comment by pocka** [comment]
thanks, turned out it's a typo :frowning:

```diff
- skipIfNoParameterOrOptions
+ skipIfNoParametersOrOptions
```

**Comment by pocka** [comment]
Fixed in [v5.1.2](https://github.com/pocka/storybook-addon-designs/releases/tag/v5.1.2)

**Comment by Romanchuk** [comment]
Thanks, now it works!
